features/latest_version_features/patch_job_state_success.feature

```diff
@@ -82,6 +82,33 @@ Feature: Patch job state - Success
             | last_updated           | now or earlier            |
             | reindex_started        | now or earlier            |
             | state                  | in-progress               |
+
+    Scenario: Request is made where If-Match header is sets `*` to ask the API to ignore the ETag check
```

Comment:
This would be more consistent with the scenarios above it:
```suggestion
    Scenario: Request is made where If-Match header is set to IfMatchAnyETag (`*`) to ask the API to ignore the ETag check
```

Reply:
Done